<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>560</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="plotTypeBox">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>70</height>
      </size>
     </property>
     <property name="title">
      <string>Plot type</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="dotRadio">
        <property name="text">
         <string>Dot plot</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="histRadio">
        <property name="text">
         <string>Histogram</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="axisOptionBox">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>180</height>
      </size>
     </property>
     <property name="title">
      <string>Axes scale</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QGroupBox" name="xAxisOptionBox">
        <property name="title">
         <string>X-Axis</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item>
          <widget class="QRadioButton" name="xLinRadio">
           <property name="text">
            <string>linear</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="xLogRadio">
           <property name="text">
            <string>log</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="xLogicleRadio">
           <property name="text">
            <string>logicle</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="yAxisOptionBox">
        <property name="title">
         <string>Y-Axis</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item>
          <widget class="QRadioButton" name="yLinRadio">
           <property name="text">
            <string>linear</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="yLogRadio">
           <property name="text">
            <string>log</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="yLogicleRadio">
           <property name="text">
            <string>logicle</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="xRangeBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>61</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>70</height>
      </size>
     </property>
     <property name="title">
      <string>X-Axis range</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <widget class="QCheckBox" name="xlimAutoCheck">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Auto</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="xlimMinEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>40</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string notr="true">0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>min:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>20</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>max:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="xlimMaxEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>40</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="yRangeBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>61</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>70</height>
      </size>
     </property>
     <property name="title">
      <string>Y-Axis range</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <widget class="QCheckBox" name="ylimAutoCheck">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Auto</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="ylimMinEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>40</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string notr="true">0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>min:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>20</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>max:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ylimMaxEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>40</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="normBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>95</height>
      </size>
     </property>
     <property name="title">
      <string>Y-Axis in histogram</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <widget class="QRadioButton" name="norm2PercRadio">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>110</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Percentage</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="norm2TotalRadio">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>165</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Percentage of total</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="norm2CountRadio">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>30</y>
        <width>103</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Cell count</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="smoothBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Smooth histogram</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QSlider" name="smoothSlider">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximum">
         <number>128</number>
        </property>
        <property name="singleStep">
         <number>8</number>
        </property>
        <property name="pageStep">
         <number>8</number>
        </property>
        <property name="tracking">
         <bool>false</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
        <property name="tickInterval">
         <number>32</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="smoothSpinBox">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="maximum">
         <number>128</number>
        </property>
        <property name="singleStep">
         <number>8</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>smoothSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>smoothSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>720</x>
     <y>764</y>
    </hint>
    <hint type="destinationlabel">
     <x>913</x>
     <y>766</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>smoothSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>smoothSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>913</x>
     <y>766</y>
    </hint>
    <hint type="destinationlabel">
     <x>720</x>
     <y>764</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>histRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>smoothBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1357</x>
     <y>594</y>
    </hint>
    <hint type="destinationlabel">
     <x>1297</x>
     <y>998</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xlimAutoCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>xlimMaxEdit</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1242</x>
     <y>758</y>
    </hint>
    <hint type="destinationlabel">
     <x>1387</x>
     <y>759</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xlimAutoCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>xlimMinEdit</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1242</x>
     <y>758</y>
    </hint>
    <hint type="destinationlabel">
     <x>1317</x>
     <y>759</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ylimAutoCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>ylimMinEdit</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1242</x>
     <y>826</y>
    </hint>
    <hint type="destinationlabel">
     <x>1317</x>
     <y>827</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ylimAutoCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>ylimMaxEdit</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1242</x>
     <y>826</y>
    </hint>
    <hint type="destinationlabel">
     <x>1387</x>
     <y>827</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dotRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>normBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>172</x>
     <y>448</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>histRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>normBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>172</x>
     <y>448</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
